corehq/ex-submodules/casexml/apps/phone/tests/test_ota_restore.py

```diff
@@ -105,19 +107,17 @@ def tearDown(self):
 @override_settings(CASEXML_FORCE_DOMAIN_CHECK=False)
 class OtaRestoreTest(BaseOtaRestoreTest):
 
+    def _get_the_only_synclog(self):
```

Comment:
Since this is called `get_the_ONLY_synclog` should it raise an exception if there are more than one? Otherwise maybe it should be called `get_the_first_synclog`?

Reply:
These tests are written such a way that there would be only one synclog, and the query was relying on that fact, so exception is necessary I will change the name.